pinot-query-runtime/src/test/resources/queries/NullHandling.json

```diff
@@ -0,0 +1,149 @@
+{
+  "tbl_level_null_handling": {
```

Comment:
Can you also add some plan-based tests in pinot-query-planner?

Particularly for anti semi-join queries such as: `SELECT .. FROM T1 WHERE uuid NOT IN (SELECT uuid FROM ...)`

If the uuid is column is nullable, this gets converted to a broadcast join (iirc).

Would be great if we can add test-cases for these.

Reply:
added planner test